infra/gcp/ensure-staging-storage.sh

```diff
@@ -164,3 +173,31 @@ for REPO; do
 
     color 6 "Done"
 done
+
+# Special case: Release Managers
+for repo in ${RELEASE_STAGING_PROJECTS[*]}; do
```

Comment:
SUPA nit:

`"${RELEASE_STAGING_PROJECTS[@]}"` - it doesn't matter much here, but it's the "safe" way to do arrays wrt whitespace

Reply:
Fixed.